gps_general.h

```diff
@@ -3,15 +3,11 @@
 
 #include <xc.h>
 
-#define _XTAL_FREQ 1000000
+#define _XTAL_FREQ 48000000
 #define MAX_LOOP_TIME_DIFF_ms 500
 
-#define LED_1_ON() (LATB1 = 0)
-#define LED_1_OFF() (LATB1 = 1)
-#define LED_2_ON() (LATB2 = 0)
-#define LED_2_OFF() (LATB2 = 1)
-#define LED_3_ON() (LATB3 = 0)
-#define LED_3_OFF() (LATB3 = 1)
+#define LED_1_LAT LATB1
+#define LED_2_LAT LATB2
```

Comment:
ok I'll change it back. I personally like using LAT because it reduces code clutter, and imo `LED_LAT = 0` reads the same as `LED_OFF()`. (writing low vs turning of sounds like the same thing). But yeah I'll change it.